File: Modul_6/Soal_3/PRAK603-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);
    int angka1[n1], angka2[n2];
    if(n1 == n2) {
        for(int i = 0; i < n1; i++) {
            scanf("%d", &angka1[i]);
        }
        for(int i = 0; i < n2; i++) {
                scanf("%d", &angka2[i]);
                printf("%d ", angka1[i] * angka2[i]);
        }        
    }
    else {
        printf("Jumlah tidak sama");
    }
}

File: Modul_6/Soal_3/PRAK603-2310817210015-MuhammadBukhariFitri.py
n1, n2 = map(int, input().split())
matriks1 = []
matriks2 = []
if (n1 == n2):
    angka1 = list(map(int, input().split()))
    angka2 = list(map(int, input().split()))

    for i in range (n1):
        matriks1.append(angka1[i])
    for i in range (n2):
        matriks2.append(angka2[i])
    for i in range (n1):
        print(f"{matriks1[i] * matriks2[i]}", end = " ")
        

else:
    print("Jumlah tidak sama")